met/src/libcode/vx_data2d_nccf/nccf_file.cc

```diff
@@ -2568,7 +2604,7 @@ bool NcCfFile::get_grid_from_coordinates(const NcVar *data_var) {
       is_x_dim_var = is_y_dim_var = false;
       for (int cIdx = 0; cIdx<count; cIdx++) {
         if ( Var[var_num].name == sa[cIdx]) {
-          if (get_nc_att(Var[var_num].var, (string)"units", units_value)) {
+          if (get_nc_att_value(Var[var_num].var, (string)"units", units_value)) {
```

Comment:
Howard, here's the line where that warning is originating. This function is parses information about the grid but doesn't print any info about that! I'd recommend...
(1) add a debug(4) log message stating the name of the variable which defines the latitude.
(2) add a debug(4) log message stating the name of the variable which defines the longitude.
(3) change the warning on line 2617 to a debug(4) message stating that that coordinate variable name does not define the grid.